pkg/build/oci/oci.go

```diff
@@ -213,9 +233,26 @@ func buildImageFromLayerWithMediaType(mediaType ggcrtypes.MediaType, layerTarGZ
 
 func Copy(src, dst string) error {
 	log.DefaultLogger().Infof("Copying %s to %s", src, dst)
-	if err := crane.Copy(src, dst, crane.WithAuthFromKeychain(keychain)); err != nil {
+	srcRef, err := name.ParseReference(src)
+	if err != nil {
+		return err
+	}
+	dstRef, err := name.ParseReference(dst)
+	if err != nil {
+		return err
+	}
+	desc, err := remote.Get(srcRef, remoteOpts...)
+	if err != nil {
+		return fmt.Errorf("fetching %s: %w", src, err)
+	}
+	pusher, err := remote.NewPusher(remoteOpts...)
+	if err != nil {
+		return err
+	}
+	if err := pusher.Push(context.TODO(), dstRef, desc); err != nil {
```

Comment:
I think it's fine to change the signature of `Copy`, if that's why you're going with `context.TODO()` here. It's pre-1.0, and I don't expect anybody should be using this in apko anyway, vs using something like `crane.Copy` directly.

Reply:
Plumbed it around.